app/views_redesigned/publisher_dashboards/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :actions do %>
+  <%= render "/shared/date_range_picker" %>
+<% end %>
+
+<% content_for :page_heading do %>
+  <%= page_heading "Publisher", "Dashboard",
+    subtitle: "Overall view of your account",
+    icon: "fas fa-tachometer" %>
+<% end %>
+
+<main id="content" role="main" class="bg-light">
+  <div class="container space-1">
+    <div class="card-deck d-block d-lg-flex card-lg-gutters-3 mb-3">
+      <%= render_later async_publisher_impressions_count_stat_card_path do %>
+        <i class="fa fa-spinner fa-spin text-primary"></i>
+      <% end %>
+      <%= render_later async_publisher_clicks_count_stat_card_path do %>
+        <i class="fa fa-spinner fa-spin text-primary"></i>
+      <% end %>
+      <%= render_later async_publisher_click_rate_stat_card_path do %>
+        <i class="fa fa-spinner fa-spin text-primary"></i>
+      <% end %>
+      <%= render_later async_publisher_revenue_stat_card_path do %>
+        <i class="fa fa-spinner fa-spin text-primary"></i>
+      <% end %>
+    </div>
+
+    <div class="text-center mb-3 mt-5">
+      <span class="u-divider u-divider--xs u-divider--text">Active Properties</span>
+    </div>
+
+    <div class="mb-3">
+      <div class="card-deck d-block d-md-flex card-sm-columns card-sm-2-count card-md-3-count">
```

Comment:
can you make sure `card-sm-columns` is a valid class? I think that came from the old theme.